src/Entity/ThematicCommunity/AdherentMembership.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Entity\ThematicCommunity;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use App\Entity\Adherent;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use libphonenumber\PhoneNumber;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="thematic_community_membership_adherent")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class AdherentMembership extends ThematicCommunityMembership
+{
+    /**
+     * @var ThematicCommunity[]|Collection
+     *
+     * @ORM\ManyToMany(targetEntity="App\Entity\ThematicCommunity\ThematicCommunity")
```

Comment:
1/ 
`ManyToOne`, no?

`Adherent (1) <-> (n) AdherentMembership`
and 
`AdherentMembership (n) <-> (1) ThematicCommunity`

2/ could be moved into `ThematicCommunityMembership` as it is the same relation for all types of Membership